### ISSUE TITLE
how to reduce the sampling speed?

### ISSUE THREAD

**Issue by crboffo** [garbled1/balboa_homeassistan#20]
Hi, 
I noticed that when this integration is running, the access through the standard app becomes very difficult. Could this be connected to teh fact that the integrations keeping the "channel" occupied?

**Comment by garbled1** [comment]
I do not think you can use this integration, and the app at the same time.  Both of these connect to the port on the device (which is extremely stupid) and block the other from working.  I cannot even connect to the port from 2 different machines at the same time.  Additionally, the sample rate is pushed from the wifi device, not requested, so it's not even a matter of rate setting, the device itself just spews data.

It's a very poorly made device, I did the best I could with it.  Sorry :(
